server/schedule/operator.go

```diff
@@ -39,6 +39,10 @@ const (
 	RegionOperatorWaitTime = 10 * time.Minute
 	// RegionInfluence represents the influence of a operator step, which is used by ratelimit.
 	RegionInfluence int64 = 1000
+	// smallRegionInfluence represents the influence of a operator step, which is used by ratelimit.
```

Comment:
The comment here is the same as `RegionInfluence`, so what is the difference?

Reply:
Do we need to consider the empty region?

Reply:
Updated @shafreeck 
I think we don't need to consider the empty region. Once the region is empty (e.g. the size will be 1MB), the behavior is just like before we introduce the store limit. @nolouch